lib/context_adapter_server.js

```diff
@@ -274,6 +319,11 @@ function requestHandler(request, reply) {
 
   if (caComm.isPollingUpdateContext(request)) {
     respond2Request(request, reply);
+  } else if (caComm.getCellIdDescriptors(request).length) {
+    async.parallel([
+      respond2Request.bind(null, request, reply),
+      processCellIdRequest.bind(null, request)
```

Comment:
They are not 100% independent, as long as they both are given a reference to the same object. Imagine one of them do something like:
```
 delete request.body
```
What will happen if the other one needed that field? Oh, immutability, where art thou?

Anyway, if you think that's safe, I'll gladly believe you. If that's a problem, it will arise in Xavi's tests.